themes/src/main/resources/theme/keycloak.v2/login/template.ftl

```diff
@@ -92,10 +92,13 @@
             "${url.ssoLoginInOtherTabsUrl?no_esc}"
         );
     </script>
+    <script>
+      // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1404468
+      const isFirefox;
```

Comment:
How does this fix the issue exactly? There are plenty of script being loaded above already.

Reply:
I don't know how, I haven't looked at how firefox works, all I can say is that this removes the initial un-styled view

Reply:
I guess it just adds a blocking script to the page, the contents contents could be anything. What is the root cause of this issue? If it is a Firefox bug I would not bother fixing it with a hack, that is the responsibility of the Firefox team.